### ISSUE TITLE
Add graal benchmarks

### ISSUE THREAD

**Issue by nicolasstucki** [liufengyun/bench#142]
We should have a profile for compiling dotty with dotty tuning on Graal CE and Graal EE.

**Comment by liufengyun** [comment]
It's possible to do that, except it will double test time from 3 hours to 6
hours. This will have a big impact development.

Le jeu. 17 mai 2018 18:16, Nicolas Stucki <notifications@github.com> a
écrit :

**Comment by nicolasstucki** [comment]
Graal seams to be 40% faster. It should not be that much. We can start with the EE and see what happens.

**Comment by liufengyun** [comment]
If the purpose is to test Graal, we can have a separate machine, run only
on Graal.

Le jeu. 17 mai 2018 18:29, Nicolas Stucki <notifications@github.com> a
écrit :

**Comment by nicolasstucki** [comment]
Yes, that could work as long as we also execute the same benchmark on hotspot in that machine.
